ansible/roles/fes-app-config/files/deployment-scripts/frontend_deployment.yml

```diff
@@ -1,4 +1,25 @@
 ---
+  - name: CA Certs
+    hosts: localhost
+    gather_facts: true
+    vars:
+      cert_name: "chca"
+    tasks:
+      - name: Download the cert artifact from S3
+        aws_s3:
+          bucket: "{{ s3_bucket_releases }}"
+          object: "chl-fes/{{ cert_name }}.pem"
+          dest: "/etc/pki/ca-trust/source/anchors/{{ cert_name }}.pem"
+          mode: get
+
+      - name: Update trust
+        shell: /usr/bin/update-ca-trust
+      
+      - name: Force systemd to read all configs
+        systemd:
+          daemon_reload: yes
```

Comment:
Should use notify if you need to run restarts or systemd commands, just means they wait until the end of the playbook to run them altogether but keeps the code cleaner.

Reply:
changed to use a notify handler